polyfill/lib/calendar.mjs

```diff
@@ -1812,7 +1801,8 @@ const helperChinese = ObjectAssign({}, nonIsoHelperBase, {
         if (numberPart[0] === '0') numberPart = numberPart.slice(1);
         let monthInfo = months[numberPart];
         month = monthInfo && monthInfo.monthIndex;
-        // If this leap month isn't present in this year, constrain down to the last day of the previous month.
+        // If this leap month isn't present in this year, constrain to the same
+        // day of the previous month.
```

Comment:
Same comment as above: why not the last day of the previous month, because it's the closest to the missing date?

Reply:
Same here, based on the discussion on Friday I'm assuming this is OK.